scripts/azure-pipelines/build_linux.bash

```diff
@@ -7,7 +7,22 @@
 
 ver=$(python scripts/mumble-version.py)
 
-qmake -recursive CONFIG+="release tests warnings-as-errors" DEFINES+="MUMBLE_VERSION=${ver}"
+# Use the Qt kit installed through PPA
+# even though the source executes successfully, it does return code 1
+source /opt/qt*/bin/qt*-env.sh || true
+
+# clone and install AppImageUpdaterBridge to the system
+git clone https://github.com/antony-jr/AppImageUpdaterBridge
+cd AppImageUpdaterBridge
+git checkout "v1.1.6" 
+cmake -DLOGGING_DISABLED=ON .
+make -j $(nproc)
+make install
```

Comment:
I think the Linux CI is currently failing because you'll have to use `sudo` for this one